projects/ids-enterprise-ng/src/lib/calendar/soho-calendar.component.ts

```diff
@@ -15,6 +15,19 @@ import {
   Output,
 } from '@angular/core';
 
+/********************************************************************
+ * Calendar Week View Element
+ *******************************************************************/
+// @ts-ignore
+@Component({
+  selector: 'div[soho-calendar-weekview]', // tslint:disable-line
+  template: `<ng-content></ng-content>`,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SohoCalendarWeekViewComponent { // implements AfterViewInit, OnDestroy {
```

Comment:
I removed the commented code and merged with master again today, hopefully the build is now clean. Thanks for noticing the bug, this is also observed on the enterprise project when switching from month view to week view to day view, I will create a git issue for @tmcconechy .

Reply:
Yeah just need the steps must be something i missed as that behavior is in the enterprise component.

Reply:
I would encourage us to practice rebase instead of merging the latest master. Ideally we would want  [a linear git history](https://www.bitsnbites.eu/a-tidy-linear-git-history/) to benefit from better backtracking and maintainability.

Also please link the enterprise ticket that we need here. Or do we want this feature out now and compromise to roll this out without that enterprise fix?

Reply:
I agree with rebasing when you can.
Git ticket is https://github.com/infor-design/enterprise/issues/3319 , I wont be able to get to it soon though and it seems non-critical so i would not hold this up.

Reply:
I did a rebase and merge. infor-design/enterprise#3319 should not hold up this PR as the bug already exists in enterprise project, without this PR, our project and others would not be able to consume the weekview component.

Reply:
@vonnyw I'm afraid it has not been rebased properly, it if did then there's no need to do merge and this PR would just contain 1 or 2 of your commits and no merge commits.